src/docs/common_tasks/globs.md

```diff
@@ -6,29 +6,33 @@ You're creating a target definition for library, binary, or other target and wan
 
 ## Solution
 
-Instead of listing files one by one, e.g. `sources=['file1', 'file2', 'file3']`, you can specify either a `globs` or an `rglobs`.
+Instead of listing files one by one, e.g. `sources=['file1', 'file2', 'file3']`, you can directly 
+use globs like `*.java`. To exclude certain files or globs, you can prefix the value with `!`, like `!ignore.java`.
```

Comment:
This reads so much better, _plus_ doesn't directly mention recursive globs, which makes me very happy.